---
layout: default
title: About
permalink: /about/
---
**under construction**

Ths Gibson Lab is part of the [Division of Computational Pathology](https://comp-path.bwh.harvard.edu/) at [Brigham and Women's Hospital](https://www.brighamhealth.org/) and [Harvard Medical School](https://hms.harvard.edu/). We are physically located in the  [Hale Building for Transformative Medicine (BTM) in 8002](https://www.google.com/maps/place/Building+for+Transformative+Medicine+at+Brigham+and+Women's+Hospital/@42.3353661,-71.1087175,15z/data=!4m5!3m4!1s0x0:0x35376a566e389c7d!8m2!3d42.3353661!4d-71.1087175) in Boston with space also at the [MIT Stata Center (32-G570)](https://www.google.com/maps/place/Stata+Center,+32+Vassar+St,+Cambridge,+MA+02139/@42.3616095,-71.0928242,17z/data=!3m1!4b1!4m5!3m4!1s0x89e370a95d3025a9:0xb1de557289ff6bbe!8m2!3d42.3616095!4d-71.0906355)  in Cambridge.

In our research we leverage tools from machine learning and control to understand biological systems.  Control theoretic concepts are integrated both in the design of our optimization schemes and statistical machine learning models, as well as in the design of our *in vitro* and *in vivo* experiments. Current and future projects include
* Statistical learning applied to microbial dynamics (Bayesian nonparametrics, layered latent state-space models)
* Theoretical foundations at the intersection of machine learning and control
* Genetically engineering and controlling a microbial consortia
* Host-Microbiome interactions
* Time-series "omics" (metagenomics, transcriptomics, and metabolomics)


## Sample from our recent theoretical learning research
<figure class="image" style="
    float: center;
    width: 450px;
    margin: auto;
    display: block;
    max-width: 100%;
    max-height: 100%;">
<img src='/image/highordertuner3.svg' alt="higher order tuner" style="width: 450px;
    display: block;
    margin: auto;" />
    <figcaption> Figure 1: Higher order tuner demonstrating the ability to maintain stability during model training <button name="button" onclick="window.location.href='https://arxiv.org/abs/2005.01529'"><i class="far fa-file-pdf"></i> paper</button></figcaption>
</figure>
## Sample from our recent experimental biology research
Our main experimental system is gnotobiotic mice (germ-free), where we can colonize mice with whatever background flora we desire.

<figure class="image">
<img src='/image/mouse_time.png' alt="mouse time series" style="width: 650px;
    display: block;
    margin: auto;" />
    <figcaption> Figure 2: Mouse time series <button name="button" onclick="window.location.href='/papers/hsu19hostmicrobe.pdf'"><i class="far fa-file-pdf"></i> paper</button></figcaption>
</figure>

## Samples from our modeling research

<figure class="image" style="float: right; width: 375px;">
<a href="/posters/gibson18icml.pdf"><img src='/image/gibson18icml.png' alt="mouse time series" style="width: 375px;
    display: block;
    margin: auto;
    float: auto;" /></a>
    <figcaption> Figure 3: Robust and Scalable Models of Microbiome Dynamics
    <button name="button" onclick="window.location.href='http://proceedings.mlr.press/v80/gibson18a.html'"><i class="far fa-file-pdf"></i> paper</button>
    <button name="button" onclick="window.location.href='/posters/gibson18icml.pdf'"><i class="far fa-file-pdf"></i> poster</button>
    </figcaption>
</figure>

<figure class="image" style="float: left;">
<a href="/posters/kim20mlcb.pdf"><img src='/image/kim20mlcb.png' alt="mouse time series" style="width: 375px;
    display: block;
    margin: auto;
    float: auto;" /></a>
    <figcaption> Figure 4: Chronostrain
    <button name="button" onclick="window.location.href='/posters/kim20mlcb.pdf'"><i class="far fa-file-pdf"></i> poster</button>
    </figcaption>
</figure>


## Current collaborative projects

[National Science Foundation - The rules of microbiota colonization of the mammalian gut](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2025515&HistoricalAwards=false)







![The rules of microbiota colonization of the mammalian gut](/image/nsfmtm.svg){: style="
    display: block;
    margin: auto;"}

>The goal of the project is to derive fundamental rules that describe and predict the dynamic process of microbial colonization of the mammalian gut. To achieve this goal, the team of investigators will develop new computer-based methods to automatically extract predictive and explanatory rules from large microbiome data sets. The team will also develop new experimental tools and generate data sets in mouse measuring how microbiomes change over time and across space in the mammalian gut. Overall, the project will further the understanding of the formation of microbiomes in mammals and can provide broader insights into the emergence of other microbial ecosystems, such as those in soil and marine environments
>
>This project has three main thrusts: 1) Learn microbial community-level rules that quantitatively predict population dynamics of mouse gut colonization and assess these rules across differing ranges of microbial diversity and composition, 2) Elucidate microbial gene-level mechanisms that predict mouse gut colonization dynamics, and 3) Profile microbial spatiotemporal organization and dynamics during gut colonization at the species and gene level to predict microbial community dynamics.
