---
layout: default
title: Carsen Stringer ABC Talk
permalink: /miller_talk_2021/
---

## ***Inference in generalized bilinear models***

![Jeff Miller](http://jwmi.github.io/images/my-photo-medres-crop.jpg){: style="width: 200px;
    border-radius: 5%;
    display: block;
    margin: 10pt;
    float: right;"}


Monday May, 17th, 2021 \\
4:00-5:00 PM Eastern  \\
Zoom link <i class="fa fa-external-link"></i> <https://mit.zoom.us/j/92499692918>\\
iCal <i class="fas fa-file-download"></i> [ABC_Seminar_May_17.ics](/talks/ABC_Seminar_May_17.ics)


**Jeff Miller, PhD** \\
&nbsp;&nbsp; Assistant Professor of Biostatistics \\
&nbsp;&nbsp; Harvard T.H. Chan School of Public Health

**Abstract**: Latent factor models are widely used to discover and adjust for hidden variation in modern applications.  However, most methods do not fully account for uncertainty in the latent factors, which can lead to miscalibrated inferences such as overconfident p-values.  In this article, we develop a fast and accurate method of uncertainty quantification in generalized bilinear models, which are a flexible extension of generalized linear models to include latent factors as well as row covariates, column covariates, and interactions.  In particular, we introduce delta propagation, a general technique for propagating uncertainty among model components using the delta method.  Further, we provide a rapidly converging algorithm for maximum a posteriori GBM estimation that extends earlier methods by estimating row and column dispersions.  In simulation studies, we find that our method provides approximately correct frequentist coverage of most parameters of interest.  We demonstrate on RNA-seq gene expression analysis and copy ratio estimation in cancer genomics.  arXiv preprint: <http://arxiv.org/abs/2010.04896>
